src/CBLVectorIndexConfig_CAPI.cc

```diff
@@ -0,0 +1,48 @@
+//
+// CBLVectorIndexConfig_CAPI.cc
+//
+// Copyright (C) 2024 Couchbase, Inc All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+
```

Comment:
empty line

Reply:
Will fix this in the later PR as don't want to wait for another round of PR validation.